app/views/asylum_support_decisions/_form.html.erb

```diff
@@ -5,7 +5,7 @@
     <%= render partial: "shared/form_preview" %>
 
     <%= f.select :tribunal_decision_category, f.object.facet_options(:tribunal_decision_category), { label: "Category" }, { class: 'form-control' } %>
-    <%= f.select :tribunal_decision_sub_category, f.object.facet_options(:tribunal_decision_sub_category), { label: "Sub-category" }, { class: "form-control", data: { placeholder: "Select sub-category when relevant" } } %>
+    <%= f.select :tribunal_decision_sub_category, [""] + f.object.facet_options(:tribunal_decision_sub_category), { label: "Sub-category" }, { class: "form-control", data: { placeholder: "Select sub-category when relevant" } } %>
```

Comment:
Minor, but passing `include_blank: true` to the options for `f.select` would be a more "rails-y" way of doing this.